seqr/views/apis/data_manager_api_tests.py

```diff
@@ -756,36 +756,53 @@ def _set_file_iter_stdout(rows):
                 self.assertEqual(model_cls.objects.count(), params['initial_model_count'])
                 mock_send_slack.assert_not_called()
 
+                def _test_basic_data_loading(data, samples, projects, project_names, individual_id):
```

Comment:
did anything in this block change, or is it just copy-pasted? It might be better to do the refactor as a separate PR with this branched off of it to make it really clear what is changing in the tests

Reply:
Yes, they are copy-pasted from the loading new data test. I can create a PR for it. The basic test function is necessary because the additional data loading tests can do a partial test requiring little effort.